react/features/shared-video/components/web/AbstractVideoManager.ts

```diff
@@ -342,54 +343,93 @@ class AbstractVideoManager extends PureComponent<IProps> {
     /**
      * Seeks video to provided time.
      *
-     * @param {number} time
+     * @param {number} _time - Time to seek to.
+     * @returns {void}
      */
-    seek: (time: number) => void;
+    seek(_time: number) {
+        // to be implemented by subclass
+    }
 
     /**
      * Indicates the playback state of the video.
+     *
+     * @returns {string}
      */
-    getPlaybackStatus: () => string;
+    getPlaybackStatus(): string | undefined {
+        return;
+    }
 
     /**
      * Indicates whether the video is muted.
+     *
+     * @returns {boolean}
      */
-    isMuted: () => boolean;
+    isMuted(): boolean | undefined {
+        return;
+    }
 
     /**
      * Retrieves current volume.
+     *
+     * @returns {number}
      */
-    getVolume: () => number;
+    getVolume(): number | undefined {
```

Comment:
Can't we make this return a number always, to avoid all the calls to Number(foo) above?